BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -442,6 +442,24 @@ extern NSString *const BOXAPIEnterpriseEventTypeCollaborationExpiration;
 extern NSString *const BOXAPIEnterpriseEventTypeItemSync;
 extern NSString *const BOXAPIEnterpriseEventTypeItemUnsync;
 
+//urlsessiontask cache dir, file prefix
+extern NSString *const BOXURLSessionTaskCacheDirName;
```

Comment:
Minor: would prefer Dir to expand to Directory for clarity. Also, proper caps and space after // for consistency.